composer/profiler/profiler.py

```diff
@@ -23,20 +23,6 @@
 log = logging.getLogger(__name__)
 
 
-@dataclasses.dataclass
-class ProfilerEventHandlerHparams(hp.Hparams, abc.ABC):
-    """Base class for profile event handler hparams."""
-
-    @abc.abstractmethod
-    def initialize_object(self) -> ProfilerEventHandler:
-        """Constructs and returns an instance of the :class:`ProfilerEventHandler`.
-
-        Returns:
-            ProfilerEventHandler: The event handler.
-        """
-        pass
-
-
 class ProfilerEventHandler(Callback, abc.ABC):
```

Comment:
In addition, can we update the signatures for `process_duration_event`, `process_instant_event`, and `process_counter_event` to take `Timestamp` objects instead of `epoch` and `step`?

https://github.com/mosaicml/composer/blob/fef33aa9fdcde3873597a5ab108e9d26fca433ca/composer/core/time.py#L496)

Reply:
Just a thought, does it make sense to add `wall_clock_time_ns` to the `Timestamp` object?  Makes sense that wall clock time would also be tracked as part of our time abstraction.

Reply:
Yup, see #256.